pkg/apis/local/v1/types.go

```diff
@@ -54,9 +54,11 @@ type StorageClassDevice struct {
 	// StorageClass name to use for set of matched devices
 	StorageClassName string `json:"storageClassName"`
 	// Volume mode. Raw or with file system
+	// + optional
 	VolumeMode PersistentVolumeMode `json:"volumeMode"`
```

Comment:
if it is optional, should it have `omitempty`?

Reply:
fixed